app/src/main/java/com/example/android/sunshine/sync/SunshineSyncTask.java

```diff
@@ -50,6 +53,7 @@ synchronized public static void syncWeather(Context context) {
 
             /* Use the URL to retrieve the JSON */
             String jsonWeatherResponse = NetworkUtils.getResponseFromHttpUrl(weatherRequestUrl);
+            Log.v(TAG, "JSON Payload was" + jsonWeatherResponse);
```

Comment:
This does technically work, but I think it would be better to put the log statement inside of getResponseFromHttpUrl().  That way the response will be logged no matter where this function is called from.